### PR TITLE
Decisions of Polkadot Update (for JAM)

### PR DESCRIPTION
**Pull request by shawntabrizi**
This updates the Decisions of Polkadot presentation to fit into the context around JAM.

Changes:

- Start with Parallel Execution and Data Sharding
    - Move up execution cores, move down wasm
- Change Wasm to "Execution Meta-Protocol"
    - We reference that our choice today is WASM, but the title allows us to evolve to PolkaVM
- Change some references to "parachains" to "Web3 Applications", where it makes sense.
-  Remove Sassafras
- Add a section about JAM
    - What is JAM (in just a few words)
    - The decisions of JAM
        - What is the same / different from Polkadot v1

### CODE REVIEW

File: syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md [hunk 1]
```diff
@@ -556,6 +556,63 @@ An invention machine.
...
+
+</div>
+
+<div>
+
+**Changed**
+
+- Upgradability of the Core Protocol
+  - Substrate -> JAM Chain
+- Parachains -> Services
```
[the remark this answers is not in the document]
Comment (Overkillus) [reply syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md:596]: Parachains are staying but there will be something new even more fundamental then them

**Review by Overkillus — DISMISSED** [review]
Minor notes and LGTM

File: syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md [hunk 2]
```diff
@@ -248,9 +240,11 @@ A visual of XCMP channels between Parachains.
 
 <div>
 
-An often overlooked problem is economic scaling of the entire blockchain ecosystem.
+Blockspace is the capacity of a blockchain to commit and finalize operations.
+
+An often overlooked problem is the economic scaling need to provide high quality blockspace.
```
[the remark this answers is not in the document]
Comment (shawntabrizi) [reply syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md:245]: uhh, i guess the best point of elaboration is to read: 

https://www.rob.tech/blog/polkadot-blockspace-over-blockchains/

> Blockspace is not a commodity but rather a class of commodities. Blockspace produced by different systems will vary in quality, availability, and flexibility. The quality of blockspace can be judged by the security guarantees that the blockchain provides - the more secure, the higher the quality. Without a supply of blockspace, applications run into congestion or downtime, leading users to experience high fees, long wait times, or front-running. Without high-quality blockspace, applications are hacked and drained: low-quality blockspace is vulnerable to 51% attacks and toxic shock. Both types of occurrences will be familiar to readers who have spent time observing the blockchain ecosystem. These characteristics of blockspace are the key factors application developers must consider when choosing where to deploy.


My point in the slide is that shared security is what makes polkadot's blockspace "high quality" versus many other chains providing lots of blockspace too, but low quality.

File: syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (shawntabrizi) [reply syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md:596]: yes, i had the same feeling when writing the line. let me think how to clarify

Comment (shawntabrizi) [inline syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md:596]:
```suggestion
- Services (as a superset of Parachains)
```

[the remark this answers is not in the document]
Comment (shawntabrizi) [reply syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md:596]:
```suggestion
- Services (as a superset of Parachains)
```

How about this?

File: syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Overkillus) [reply syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md:245]: I think see your point but I feel like there's something wrong with the way this is worded in the slides. Economic scaling is a but undefined to me 

Based my understanding what you are saying is that blockspace quality is often sacrificed or diminished to gain more scalability, right?

Would you be okay with:
```suggestion
An often overlooked problem is that when pursuing scalability it is very easy to significantly diminish the quality of blockspace.
```

**Review by Overkillus — APPROVED** [review]
lgtm

**Pull request merged**